docs/blog/2018-10-04-journey-to-the-content-mesh/index.md

```diff
@@ -0,0 +1,62 @@
+
+This series is intended to explain the emergence of a content mesh -- the infrastructure layer for a decoupled website. The content mesh stitches together content systems in a modern development environment while optimizing website delivery for performance.
+
+The previous generation of enterprise CMS offered a set of mediocre components; a generic content editing experience glued to out-of-date development and deployment environments. The emerging strategy for content management selects best-of-breed solutions, tailored to specific use-cases like e-commerce or blogging; it provides a modern framework for rapid iteration and it generates lightning fast websites out of the box.
+
+Change is being spurred by the confluence of three revolutions in how we create and consume content:
+
+- _Content management._ Monolithic CMS applications are being replaced by modular, specialized content systems.
+
+- _Development techniques._ Modern UI frameworks like React are becoming essential to rich user experiences.
+
+- _Performance._ As mobile surpasses 50% of internet traffic, high-performance is becoming a must-have, not a nice-to-have.
+
+<figure>
+  <img alt="The modularization of the CMS" height="400" src="./modular-cms-architecture.png" />
+  <figcaption>
+   The modularization of the CMS
+  </figcaption>
+</figure>
+
+These technological changes have made modern web technology both _essential_ for creating fresh, novel, and engaging digital experiences, and _more accessible_ for the enterprise.
+
+Each change primarily affects a different stakeholder --- marketing, development/IT, and the end-user, respectively.
+
+The key challenge is that without a content mesh, integrating these systems together is a lot of work. Forrester titled one of their report sections: "Warning: Headless Content Management Is For Do-It-Yourself Shops." But _with_ a content mesh, all of these systems can be brought together in a unified, low-cost, low-defect whole.
+
+<figure>
+  <img alt="Three areas of rapid innovation" height="300" src="./three-website-revolutions.png" />
+  <figcaption>
+    Three areas of rapid innovation
+  </figcaption>
+</figure>
+
+This post is the first in a series. As we continue, we’ll explore the rise of this "content mesh" -- that allows enterprises to adopt these new technologies without needing to DIY themselves.
+
+In Part 2, The Unbundling of the CMS, we’ll explore the rise of headless CMSs and best-of-breed website content and functionality systems.
+
+In Part 3, The Rise of Modern Web Development, we’ll detail the maturing of the Javascript ecosystem, the emergence of stable UI libraries like React for creating rich web technologies, and how they help website teams at scale build high-quality sites.
+
+In Part 4, Why Mobile Performance Has Become Crucial, we’ll discuss the ROI of site speed, explore two modern, complementary trends for improving web performance and how website teams can get the best of both out of the box.
+
```

Comment:
The magical 3-part pattern is great; very easy for people to remember and latch onto :D

Reply:
Thanks!